msg/de_DE.utf8/cubrid.msg

```diff
@@ -1714,12 +1714,12 @@ $set 8 MSGCAT_SET_PARSER_SEMANTIC
 292 Ungültige Verwendung von Aggregatfunktionen.
 293 Der Wert muss streng monoton wachsend werden für jede Partition.
 294 Kann nicht mit Funktion oder Filter Indizes auf Zeitzone Variante Spalten eingegeben.
-295 SELECT INTO is not allowed in subqueries.
-296 CTE name ambiguity, there are more than one CTEs with the same name: '%1$s'.
-297 Nested WITH clauses are not supported.
-298 Recursive CTE '%1$s' must be referenced directly in its recursive query.
-299 Error in CTE '%1$s': The non-recursive part and the recursive part must be connected with UNION ALL.
-300 Subquery is not allowed in DEFAULT clause.
+295 SELECT INTO ist bei Unterabfragen nicht erlaubt.
+296 CTE-Name Mehrdeutigkeit gibt es mehr als ein CTE mit dem gleichen Namen: '%1$s'.
+297 Verschachtelte WITH-Klauseln werden nicht unterstützt.
+298 Recursive CTE '%1$s' muss direkt in ihrer rekursiven Abfrage referenziert werden.
```

Comment:
Should it be?
**Rekursive** CTE '%1$s' muss direkt in ihrer rekursiven Abfrage referenziert werden.